### PR TITLE
resolved Issue #8687

### PR DESCRIPTION
**Pull request by hooman-mirghasemi**
now if in config file in the components -> formmater  set dateType to jalali. asDate and asDateTime functionns will return jalali calender date.

### CODE REVIEW

File: framework/i18n/Formatter.php [hunk 1]
```diff
@@ -89,8 +90,13 @@ class Formatter extends Component
      * @since 2.0.1
      */
     public $defaultTimeZone = 'UTC';
+    /*
+     * @var string that set the calendar type.
+     * can be "gregorian" or "jalali"
+     */
+    public $dateType = 'gregorian';
```
Comment (armpogart) [inline framework/i18n/Formatter.php]: Maybe will be better to name it `caldendarType`?

**Comment by hooman-pro** [comment]
@armpogart that's right. what should i don now? how can i update this pull request? changing in my repo and send pull request again?

File: framework/i18n/Formatter.php [hunk 2]
```diff
@@ -89,8 +90,13 @@ class Formatter extends Component
      * @since 2.0.1
      */
     public $defaultTimeZone = 'UTC';
+    /*
+     * @var string that set the calendar type.
+     * can be "gregorian" or "jalali"
+     */
+    public $calendarType = 'gregorian';
```
Comment (hooman-pro) [inline framework/i18n/Formatter.php]: I changed it

**Comment by cebe** [comment]
closing this as #8687 is already resolved. Thank you for your contribution!

**Pull request closed without merging**